web/themes/custom/contribtracker/gulp-tasks/scss.js

```diff
@@ -1,6 +1,6 @@
 import autoprefixer from 'autoprefixer';
 import postcss from 'gulp-postcss';
-import dartSass from 'sass'
+import * as dartSass from 'sass'
```

Comment:
The error pointed only to this as a problem. Is the `postcss-scss` dependency related to this?

Reply:
No this was also a warning displaying when running `gulp`
<img width="1043" alt="Screenshot 2024-05-20 at 5 52 27 PM" src="https://github.com/contrib-tracker/backend/assets/2538164/8487288d-53bc-4f10-a282-e317208fef7c">

Reply:
Yes, that is what I mean. This is the change that will fix the warning. Is the postcss change related to this?

Reply:
yes

Reply:
So, you're fixing two different issues in this PR, right? Let's avoid that in the future. Create as many PRs as you need.